### ISSUE TITLE
PAL Build of PS2 SM64 has broken audio and broken font + will not boot on Actual Hardware

### ISSUE THREAD

**Issue by Deinsti** [fgsfdsfgs/sm64-port#12]
**_Font Error:_**
![gsdx_20200923123155](https://user-images.githubusercontent.com/50426480/94007281-258c5980-fd99-11ea-8a6b-84f3b2dbf416.png)
![gsdx_20200923123210](https://user-images.githubusercontent.com/50426480/94007285-2624f000-fd99-11ea-8807-3e8bb9459686.png)
![gsdx_20200923123301](https://user-images.githubusercontent.com/50426480/94007286-26bd8680-fd99-11ea-8065-f0d96e9a4ac9.png)
**_Won't boot on Actual Hardware (video):_**

[HwVideoEditor_2020_09_23_123802.zip](https://github.com/fgsfdsfgs/sm64-port/files/5267863/HwVideoEditor_2020_09_23_123802.zip)

As shown above, the issue's I would like to report is when building the PAL version of the SM64 PS2 ELF

- The font heavily breaks; I assume the code hasn't been adapted to support the German and French script but that could be wrong

- The Audio doesn't work 99% of the time, the reason I say 99% is because there are VERY brief moments like when Mario head appears at the start - the audio plays for a fraction of a second and then stops and nothing else plays - which suggests that the Audio server does function; it's just fails to play the tracks

- The last issue which I wanted to report is that the PAL Build simply doesn't boot on a real PAL PS2 (FreeMCBoot), the footage for that has been linked in the zip file at the end

While running in the PSCX2 emu, the PAL ELF does throw out these errors in the Log however I don't know which error they relate to: `(EE pc:001AC14C) TLB Miss, addr=0x23e [load]`
`(EE pc:001AC168) TLB Miss, addr=0x23e [load]`
`(EE pc:001AC168) TLB Miss, addr=0x0 [load]`
`(EE pc:001AC17C) TLB Miss, addr=0x22a [load]`
`(EE pc:001AC18C) TLB Miss, addr=0x4c [load]`
`(EE pc:001AC18C) TLB Miss, addr=0x44 [load]`
`(EE pc:001A94B8) TLB Miss, addr=0x48 [load]`
`(EE pc:001A94B8) TLB Miss, addr=0x4c [load]`
`(EE pc:001A94B8) TLB Miss, addr=0x50 [load]`
`(EE pc:001A94B8) TLB Miss, addr=0x54 [load]`
`(EE pc:001AC908) TLB Miss, addr=0x88 [load]`
`(EE pc:001AC908) TLB Miss, addr=0x98 [load]`
`(EE pc:001AC908) TLB Miss, addr=0xa8 [load]`
`(EE pc:001AC908) TLB Miss, addr=0xb8 [load]`
`(EE pc:001A954C) TLB Miss, addr=0x0 [load]`
`(EE pc:001A954C) TLB Miss, addr=0x0 [store]`
`(EE pc:001AC3E8) TLB Miss, addr=0x22a [store]`
`(EE pc:001AC14C) TLB Miss, addr=0x2f2 [load]`
`(EE pc:001AC168) TLB Miss, addr=0x2f2 [load]`
`(EE pc:001AC168) TLB Miss, addr=0x0 [load]`
`(EE pc:001AC17C) TLB Miss, addr=0x2de [load]`
`(EE pc:001AC18C) TLB Miss, addr=0x4c [load]`
`(EE pc:001AC18C) TLB Miss, addr=0x44 [load]`
`(EE pc:001A94B8) TLB Miss, addr=0x48 [load]`
`(EE pc:001A94B8) TLB Miss, addr=0x4c [load]`
`(EE pc:001A94B8) TLB Miss, addr=0x50 [load]`
`(EE pc:001A94B8) TLB Miss, addr=0x54 [load]`
`(EE pc:001AC908) TLB Miss, addr=0x88 [load]`
`(EE pc:001AC908) TLB Miss, addr=0x98 [load]`
`(EE pc:001AC908) TLB Miss, addr=0xa8 [load]`
`(EE pc:001AC908) TLB Miss, addr=0xb8 [load]`
`(EE pc:001A954C) TLB Miss, addr=0x0 [load]`
`(EE pc:001A954C) TLB Miss, addr=0x0 [store]`
`(EE pc:001AC3E8) TLB Miss, addr=0x2de [store]`
`(EE pc:001AC14C) TLB Miss, addr=0x3a6 [load]`
`(EE pc:001AC168) TLB Miss, addr=0x3a6 [load]`
`(EE pc:001AC168) TLB Miss, addr=0x0 [load]`
`(EE pc:001AC17C) TLB Miss, addr=0x392 [load]`
`(EE pc:001AC18C) TLB Miss, addr=0x4c [load]`
`(EE pc:001AC18C) TLB Miss, addr=0x44 [load]`
`(EE pc:001A94B8) TLB Miss, addr=0x48 [load]`
`(EE pc:001A94B8) TLB Miss, addr=0x4c [load]`
`(EE pc:001A94B8) TLB Miss, addr=0x50 [load]`
`(EE pc:001A94B8) TLB Miss, addr=0x54 [load]`
`(EE pc:001AC908) TLB Miss, addr=0x88 [load]`
`(EE pc:001AC908) TLB Miss, addr=0x98 [load]`
`(EE pc:001AC908) TLB Miss, addr=0xa8 [load]`
`(EE pc:001AC908) TLB Miss, addr=0xb8 [load]`
`(EE pc:001A954C) TLB Miss, addr=0x0 [load]`
`(EE pc:001A954C) TLB Miss, addr=0x0 [store]`

I understand that the priority will be to get the US version working fully first but I wanted to just report on existing issues should the PAL version get more attention

**Comment by fgsfdsfgs** [comment]
EU audio code is wack in any version of sm64-port, there's definitely some funky undefined behavior things going on.

**Comment by Freddypicapiedra** [comment]
@fgsfdsfgs 

Good afternoon

Is it possible to improve the graphics in a future update of the SM64 PORT PS2 on the Amigo console hardware?

I admire you very much, you are doing an excellent job.

Regards

**Comment by fgsfdsfgs** [comment]
Amigo?

**Comment by Freddypicapiedra** [comment]
@fgsfdsfgs 
Genius, With all due respect

**Comment by fgsfdsfgs** [comment]
I don't know what SenSei is either.

**Comment by Freddypicapiedra** [comment]
@fgsfdsfgs 

a way of saying teacher, genius, smart, admiration

**Comment by fgsfdsfgs** [comment]
No, I mean what exactly did you mean by "work on SenSei console hardware"? If you just mean a real PS2, it does work on a real PS2. If you mean specifically the EU version, unfortunately so far I have no idea what causes the crashes in there.

**Comment by Freddypicapiedra** [comment]
@fgsfdsfgs 

I mean if the sm64 could be improved a bit so that it can look like the pcsxe2 emulator.
since I saw that it has better performance in the pcsxe2
